docs/guide.rst

```diff
@@ -196,31 +196,116 @@ the scene as desired::
+are also helper objects, lights, and cameras. These are all world objects.
 
-Geometry
---------
+**Geometry**
 
-Each world object has a geometry. This geometry object contains the
+Most world objects have a geometry. This geometry object contains the
 data that defines (the shape of) the object, such as positions, plus
 data associated with these positions (normals, texcoords, colors, etc.).
 Multiple world objects may share a geometry.
 
 
-Materials
----------
+**Materials**
 
-Each world object also has a material. This material object defines the
+All visible world objects have a material. This material object defines the
 appearance of the object. Examples can be its color, how it behaves under lighting,
 what render-mode is applied, etc. Multiple world objects may share a material.
 
 
+Cameras and controllers
+-----------------------
+
+We've already been using cameras, but let's look at them a bit closer!
+
+
+**Perspective camera**
+
+There are two main cameras of interest. The first is the :class:`~pygfx.cameras.PerspectiveCamera`,
+which is a generic camera intended for 3D content. You can instantiate one
+like this:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(50, 4/3)
+
+The first argument is the fov (field of view) in degrees. This is a value between 0 and 179,
+with typical values < 100.
+
+The second argument is the aspect ratio. A window on screen is usually a rectangle
+with 4/3 or 16/9 aspect. The aspect can be set so the contents better fit the window.
+
+When the fov is zero, the camera operates in orthographic mode, where there is
+no perspective. In this case it'd be good to also specify a 3d argument: the extent
+specifies the size of the thing being viewed. So if your scene is about 400 units in size:
+
+.. code-block:: python
+
+    camera = gfx.PerspectiveCamera(0, 1, 400)
+
+
+**Orthographic camera**
+
+The second camera of interest is the :class:`~pygfx.cameras.OrthographicCamera`. It's similar
+to the perspective camera, except it's fov is always zero. It can be instantiated like this:
+
+.. code-block:: python
+
+    camera = gfx.OrthographicCamera(400, 400, maintain_aspect=False)
+
+
+The two first arguments are the `width` and `height`, which are
+typically used to initialize an orthographic camera. They have a direct
+relation with aspect and extent. (The extent is their mean, and aspect their ratio.)
+
+The `maintain_aspect` argument can be set to False if the dimensions
+do not represent a physical dimension, e.g. for plotting data. The contents
+of the view are then stretched to fill the window.
+
+
+**Orienting cameras**
+
+Camera's can be oriented manually by setting their position, and then set their rotation
+to have them look in the correct direction. This last bit can be tricky, unless you use
+the :func:`.look_at()<pygfx.cameras.PerspectiveCamera.look_at>` method. This also sets
+the extend (based on what's visible at the target location), so you can now attach
```

Comment:
```
the extent (based on what's visible at the target location), so you can now attach
```

I'm still not sure what "extent" means. Is it the near and far plane? Why does `look_at` handle that? The name implies it only affects orientation.

Reply:
This extent thing clearly needs a better explanation 😄 

It's basically the equivalent of `width` and `height` (it's the mean of these). So it's a measure for the size of the scene that the camera is to cover.

Based on this value, and the fov, the controller knows how far away from the scene the camera is, and thus e.g. around what point it should rotate. 

Since it's a measure for the size of the scene, it's a nice basis for calculating the near and far clip planes. But it *does not* directly represent the clip planes in any way.

Back to `look_at`, it works exactly as usual, except it assumes that you made the camera look at the center of the scene, so now it knows the distance between camera and scene, and from that it can calculate the extent, so that controllers can just pick up the camera and work with it (and clip planes can be calculated that likely make sense for the current scene).

Reply:
I think users will want to use `look_at` in all kinds of scenarios to update the orientation of a camera, not just to center it on the scene, and will most commonly want to manage the extent themselves. Is this new behavior optional?

Reply:
Good point, I'll see if I can work this out.